mule-deployer/src/test/java/org/mule/tools/deployment/cloudhub2/RequestBuilderTest.java

```diff
@@ -57,11 +57,11 @@ public class RequestBuilderTest {
       "   ]\n" +
       "}";
   private static final String DEPLOY_REQUEST =
-      "{\"name\":\"test-app\",\"application\":{\"ref\":{\"packaging\":\"jar\"},\"desiredState\":\"STARTED\",\"configuration\":{\"mule.agent.logging.service\":{\"artifactName\":\"test-app\",\"scopeLoggingConfigurations\":[{\"scope\":\"com.pkg.debug\",\"logLevel\":\"INFO\"}]}},\"vCores\":\"0.5\",\"integrations\":{\"services\":{\"objectStoreV2\":{\"enabled\":true}}}},\"target\":{\"targetId\":\"sampleId\",\"provider\":\"MC\",\"deploymentSettings\":{\"runtimeVersion\":\"4.2.0:v1.2.28\",\"lastMileSecurity\":false,\"clustered\":false,\"enforceDeployingReplicasAcrossNodes\":false,\"http\":{\"inbound\":{\"publicUrl\":\"test-app.mydomain.com\"}},\"forwardSslSession\":false,\"disableAmLogForwarding\":false,\"generateDefaultPublicUrl\":false,\"runtime\":{\"version\":\"4.2.0:v1.2.28\"}}}}";
+      "{\"name\":\"test-app\",\"application\":{\"ref\":{\"packaging\":\"jar\"},\"desiredState\":\"STARTED\",\"configuration\":{\"mule.agent.logging.service\":{\"artifactName\":\"test-app\",\"scopeLoggingConfigurations\":[{\"scope\":\"com.pkg.debug\",\"logLevel\":\"INFO\"}]}},\"vCores\":\"0.5\",\"integrations\":{\"services\":{\"objectStoreV2\":{\"enabled\":true}}}},\"target\":{\"targetId\":\"sampleId\",\"provider\":\"MC\",\"deploymentSettings\":{\"environment\":{\"http\":{\"inbound\":{\"publicUrl\":\"test-app.mydomain.com\"}}},\"runtimeVersion\":\"4.2.0:v1.2.28\",\"lastMileSecurity\":false,\"clustered\":false,\"enforceDeployingReplicasAcrossNodes\":false,\"http\":{\"inbound\":{\"publicUrl\":\"test-app.mydomain.com\"}},\"forwardSslSession\":false,\"disableAmLogForwarding\":false,\"generateDefaultPublicUrl\":false,\"runtime\":{\"version\":\"4.2.0:v1.2.28\"}}}}";
```

Comment:
You added {\"environment\":{\"http\":{\"inbound\":{\"publicUrl\":\"test-app.mydomain.com\"}}},\"
but it already there under : \"http\":{\"inbound\":{\"publicUrl\":\"test-app.mydomain.com\"}}